### PR TITLE
Fix: Parsing commits for changelog (fixes #5)

### LINKED ISSUE #5: Error during release calculation
The unit tests seem to work okay, but when I try to do a release using `eslint-release` (master branch), I get the following error:

```
Running tests
Hi
Hi
Calculating changes for release
C:\Users\Nicholas\projects\personal\eslint-release\lib\release-ops.js:174
            parsed[parsed.length - 1].body += log + "\n";
                                     ^
TypeError: Cannot read property 'body' of undefined
    at C:\Users\Nicholas\projects\personal\eslint-release\lib\release-ops.js:174:38
    at Array.forEach (native)
    at parseLogs (C:\Users\Nicholas\projects\personal\eslint-release\lib\release-ops.js:160:10)
    at calculateReleaseFromGitLogs (C:\Users\Nicholas\projects\personal\eslint-release\lib\release-ops.js:227:27)
    at calculateReleaseInfo (C:\Users\Nicholas\projects\personal\eslint-release\lib\release-ops.js:288:12)
    at Object.release (C:\Users\Nicholas\projects\personal\eslint-release\lib\release-ops.js:342:23)
    at Object.<anonymous> (C:\Users\Nicholas\projects\personal\eslint-release\bin\eslint-release.js:27:12)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
```

I'm not entirely sure the purpose of the code that's throwing the error. I think it's related to this change I made to fix another bug: https://github.com/eslint/eslint-release/commit/2578f31149048b413f6ea1d049eb046cb9c83b4a

Basically, you can't have a line break right in the middle of a shell command, that was causing another error.

@btmills any insights?

### PR DESCRIPTION
**Pull request by btmills**

### CODE REVIEW

**Comment by btmills** [comment]
Rebased.

File: lib/release-ops.js [hunk 1]
```diff
@@ -368,9 +368,9 @@ function release(prereleaseId) {
 
     // if there's a prerelease ID, publish under "next" tag
     if (prereleaseId) {
-        ShellOps.exec("npm publish --tag next");
+        console.log("npm publish --tag next");
```
Comment (nzakas) [inline lib/release-ops.js]: Oops?

**Comment by nzakas** [comment]
Looks like a bunch of `ShellOps.exec` calls were replaced with `console.log` calls. Probably not what you want. :)

File: lib/release-ops.js (hunk 1 above)
Comment (btmills) [inline lib/release-ops.js]: This was not supposed to go in! Haha, just a second.

**Comment by btmills** [comment]
Yeah I got smart and replaced a bunch so I could test without pushing. Looks like I missed two when I changed them back from the revert. Fixed now!

**Comment by nzakas** [comment]
Cool, I do the same thing. :) LGTM

**Pull request merged**